### PR TITLE
fix: corrected minor typo in variables.tf

### PR DESCRIPTION
**Pull request by kumar-dhanagopal**
Also ran `make docker_generate_docs` to update the README

### CODE REVIEW

**Review by tpryan — APPROVED** [review]
LGTM

**Comment by kumar-dhanagopal** [comment]
Hi @tpryan - The same tflint errors that are blocking merging of this PR appear to have been fixed by @bharathkkb in a different PR (https://github.com/GoogleCloudPlatform/terraform-google-three-tier-web-app/commit/fde60533788a72048306198e33029fc673957053). I'm guessing that re-running the failed check on that other PR will unblock this PR as well. WDYT?

**Comment by apeabody** [comment]
> Hi @tpryan - The same tflint errors that are blocking merging of this PR appear to have been fixed by @bharathkkb in a different PR ([fde6053](https://github.com/GoogleCloudPlatform/terraform-google-three-tier-web-app/commit/fde60533788a72048306198e33029fc673957053)). I'm guessing that re-running the failed check on that other PR will unblock this PR as well. WDYT?

Hi @kumar-dhanagopal, thanks for the contribution.  It does note appears that commit was merged into the main branch, however this open PR appears to address them:  https://github.com/GoogleCloudPlatform/terraform-google-three-tier-web-app/pull/15

**Pull request merged**
